src/paths/ActiveCalls.ts

```diff
@@ -12,12 +12,7 @@ export default class ActiveCalls extends PathSegment {
      *  Get Account Active (Recent) Calls
      */
     list(query?: ListQuery): Promise<PagingResult<CallLogRecord>> {
-        return this.getService().send({
-          body: undefined,
-          method: "get",
-          query: query,
-          url: this.getEndpoint(false),
-        }).then((res) => {
+        return this._send("get", false, undefined, query).then((res) => {
```

Comment:
Instead of parameters please keep using hash.

Consider having `list` and `listRaw` methods instead of ambiguity of return type of `list` in case of non-json response. ListRaw should always return ApiResponse.